rpcs3/Emu/System.cpp

```diff
@@ -1633,8 +1633,16 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 
 			if (!m_title_id.empty() && !from_hdd0_game && m_cat == "HG")
 			{
+				std::string game_dir = m_sfo_dir;
+
+				// Don't use the C00 subdirectory in our game list
+				if (game_dir.ends_with("/C00"))
```

Comment:
Assume \ is a possible delimiter on win32.

Reply:
it's added by us with + /C00

Reply:
fixed